src/coreComponents/mesh/PerforationFields.hpp

```diff
@@ -73,6 +73,14 @@ DECLARE_FIELD( wellTransmissibility,
                WRITE_AND_READ,
                "For each perforation, well transmissibility" );
 
+DECLARE_FIELD( wellSkin,
+               "wellSkin",
```

Comment:
here and in all functions names, can we use `skinFactor` instead of `skin`

Reply:
Sounds good, will change accordingly.